src/types.ts

```diff
@@ -0,0 +1,4 @@
+export enum MessageType {
```

Comment:
What's this for?

Reply:
I believe it's just a pretty way to check channel types.

Reply:
yeah this purely exists to avoid typos, it's unnecessary but could be used for events and stuff (not sure if discord.js has typed events now)